dev/config/src/webpack/webpack.config.ts

```diff
@@ -25,9 +25,9 @@ export default (mode: string) => {
 
 	return {
 		resolve: {
-			extensions: [".js", ".jsx", ".ts", ".tsx"],
+			extensions: [".js", ".jsx", ".ts", ".tsx", ".cjs"],
 			extensionAlias: {
-				".js": [".js", ".ts"],
+				".js": [".js", ".ts", ".cjs"],
```

Comment:
I think this should probably be removed cause it was just added to try and debug the webpack issue and the problem was from not finding node_modules.

Reply:
dropped